CMakeLists.txt

```diff
@@ -151,6 +151,7 @@ option(FMT_TEST "Generate the test target." ${FMT_MASTER_PROJECT})
 option(FMT_FUZZ "Generate the fuzz target." OFF)
 option(FMT_CUDA_TEST "Generate the cuda-test target." OFF)
 option(FMT_OS "Include core requiring OS (Windows/Posix) " ON)
+option(FMT_WINDOWS_NO_WCHAR "Do not use wide-char APIs, such as WriteConsoleW, on Windows" OFF)
```

Comment:
We shouldn't pollute options with such an esoteric thing. At most it could be configured with a macro. Please revert all changes to the CMake config.

Reply:
Done